src/kubectl-object-model.ts

```diff
@@ -122,6 +133,10 @@ export class Kubectl {
             return this.execute(['delete'].concat(args));
     }
 
+    public getDeployedObjects() {

```

Comment:
[Suggestion]:
- Store `deployedManifestPaths` in `kubectl` object.
- Add method `getManifestObjects(paths: string[]): any[]` in `files-helper`.
- `getDeployedObjects()` calls `getManifestObjects(this.deployedManifestPaths)` and transforms to `IKubeObject`.

This way, the objects are loaded in-memory only when actually needed. Thoughts?